client/me/concierge/shared/available-time-card.js

```diff
@@ -161,29 +285,62 @@ class CalendarCard extends Component {
 				header={ this.renderHeader() }
 			>
 				<FormFieldset>
-					<FormLabel htmlFor="concierge-start-time">
-						{ translate( 'Choose a starting time' ) }
+					<FormLabel htmlFor="concierge-start-time-group">
+						{ translate( 'Choose time of day' ) }
 					</FormLabel>
-					<FormSelect
-						id="concierge-start-time"
-						disabled={ disabled }
-						onChange={ this.onChange }
-						value={ this.state.selectedTime }
+					<Button
+						className={ btnMorningTimeGroup }
+						onClick={ () => this.handleFilterClick( 'morning' ) }
+						disabled={ morningTimes.length === 0 }
+						title={ btnMorningTitle }
 					>
-						{ times.map( ( time ) => (
-							<option value={ time } key={ time }>
-								{ this.withTimezone( time ).format( 'LT z' ) }
-							</option>
-						) ) }
-					</FormSelect>
-					<FormSettingExplanation>{ description }</FormSettingExplanation>
-				</FormFieldset>
-
-				<FormFieldset>
-					<Button disabled={ disabled } primary onClick={ this.submitForm }>
-						{ actionText }
+						<div className="shared__time-group-filter">
+							<img src={ day } alt="" />
+							{ translate( 'Morning' ) }
+						</div>
+					</Button>
+					<Button
+						className={ btnEveningTimeGroup }
+						onClick={ () => this.handleFilterClick( 'evening' ) }
+						disabled={ eveningTimes.length === 0 }
+						title={ btnEveningTitle }
+					>
+						<div className="shared__time-group-filter">
+							<img src={ night } alt="" />
+							<span>{ translate( 'Afternoon' ) }</span>
+						</div>
 					</Button>
+					<br />
+					<br />
+					{ timesForTimeGroup.length > 0 && (
```

Comment:
`timesForTimeGroup` can be empty if there are no slots in the evening for example, but since the evening filter will be disabled, you are right that we can remove this check. Done in [9d35bd2](https://github.com/Automattic/wp-calypso/pull/46999/commits/9d35bd26dfde8ccbc2434b1340d1e115012169e1).